### ISSUE TITLE
Installation instructions for Arch Linux

### ISSUE THREAD

**Issue by fluidvoice** [ElementsProject/lightning#1635]
[edit] working. See my instructions below.
Given that Arch based Manjaro is now the most popular Linux distro (Distrowatch.com) I think build and install instructions for Arch (pacman package manager) should be added to the docs.

**Comment by cdecker** [comment]
Absolutely! I don't have any experience with either distros, but I'd definitely review the docs if you were to write them up :+1:

**Comment by digoio** [comment]
Manjaro (and presumably Arch)

optionally update your system:
sudo pacman -Syyu

Get dependencies:
sudo pacman -S autoconf automake git libtool gmp sqlite python net-tools zlib gcc make

Install Bitcoin:
sudo pacman -S bitcoin-qt

For development or running tests, get additional dependencies:
sudo pacman -S asciidoc valgrind python-pip

Clone lightning:
git clone https://github.com/ElementsProject/lightning.git
cd lightning

install other python requirements
sudo pip3 install -r tests/requirements.txt

Build lightning:
./configure
make

Run bitcoind and lightning:
bitcoind &
./lightningd/lightningd &
./cli/lightning-cli help

**Comment by digoio** [comment]
On my Manjaro desktop system the first time everything built and worked OK.  Then I did the above steps in a fresh VM to verify and it did not.  Everything seemed to build OK but when I ran "./cli/lightningd-cli help" it gave the error '~/.lightning: no such file or directory'. I think the "./configure" step did not work correctly.  Am investigating.

**Comment by digoio** [comment]
OK just did it again with a fresh VM install getting same error. Did this...

# Download and install Manjaro XFCE 17.1.10
# https://manjaro.org/get-manjaro/

#skipped system update (~600MB):
#sudo pacman -Syyu

#Get dependencies:
sudo pacman -S autoconf automake git libtool gmp sqlite python net-tools zlib gcc make

#Install Bitcoin:
wget https://bitcoin.org/bin/bitcoin-core-0.16.1/bitcoin-0.16.1-x86_64-linux-gnu.tar.gz
tar -xvf bitcoin-0.16.1-x86_64-linux-gnu.tar 

#run and/or sync the bitcoin daemon testnet or mainnet
bitcoin-0.16.1/bin/bitcoind -testnet -printtoconsole

#get additional test/dev dependencies:
sudo pacman -S asciidoc valgrind python-pip

#Clone lightning:
git clone https://github.com/ElementsProject/lightning.git
cd lightning

#get other python requirements
sudo pip3 install -r tests/requirements.txt

#Build lightning:
./configure
make

#Run bitcoind and lightning:
bitcoind &
./lightningd/lightningd &
./cli/lightning-cli help

**Comment by digoio** [comment]
upon doing the step "#Build lightning" when I run ./configure I get this error...

> ./configure
> which: no pytest-3 in (/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
> which: no pytest3 in (/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
> Compiling ccan/tools/configurator/configurator...done
> Making autoconf users comfortable... yes
> checking for off_t is 32 bits... no
> checking for __alignof__ support... yes
> checking for asprintf() declaration... yes
> ...
> checking for working <ucontext.h... yes
> **checking for passing pointers via makecontext()... sh: line 1:  3190 Segmentation fault      (core dumped) ./configurator.out 2>&1**
> no
> Writing variables to config.vars.2744... yes
> Writing header to ccan/config.h... yes
> Setting PREFIX... /usr/local
> Setting CC... cc
> Setting CONFIGURATOR_CC... cc
> Setting CWARNFLAGS... -Werror -Wall -Wundef -Wmissing-prototypes -Wmissing-declarations -Wstrict-prototypes -Wold-style-definition
> Setting CDEBUGFLAGS... -std=gnu11 -g -fstack-protector
> Setting VALGRIND... 1
> Setting DEVELOPER... 0
> Setting COMPAT... 1
> Setting PYTEST... pytest
>
